packaging/linux/smoketest/README.md

```diff
@@ -0,0 +1,9 @@
+# linux smoketests
+
+setup
```

Comment:
can you make it clearer whether this is a one time per machine thing or whether this is for every release?

Reply:
(1) is obviously 1-time but unclear about the others.